test/generators/extra_pages_generator_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'pry'
+require 'test_helper'
+
+class ExtraPagesGeneratorTest < Rails::Generators::TestCase
+  tests MountainView::Generators::ExtraPagesGenerator
+
+  destination File.expand_path("../../tmp", __FILE__)
+  setup :prepare_destination
+
+  test 'Assert all views are created' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.